tests/drivers/dac/dac_api/boards/esp32s2_devkitc.overlay

```diff
@@ -0,0 +1,9 @@
+/*
+ * Copyright 2022 Espressif (Shanghai)
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&dac {
+	status = "okay";
+};
```

Comment:
Can we add this directly to the board file in order to avoid so many overlays here?

Reply:
this will case the dac is enabled by default for any application, which may increase your code size. but you can make decision.